packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -23,7 +29,9 @@ class FloatingActionButton extends StatefulComponent {
     this.elevation: 6,
     this.highlightElevation: 12,
     this.onPressed,
-    this.mini: false
+    this.mini: false,
+    this.perTabChildBuilder,
+    this.tabBarSelection // TODO: remove once this is fixed https://github.com/flutter/flutter/issues/835
   }) : super(key: key);
```

Comment:
should probably assert that you don't have both a perTabChildBuilder and a child.
Also, "perTabChildBuilder" is a mouthful of an argument name...